stonesoup/updater/pointprocess.py

```diff
@@ -0,0 +1,208 @@
+# -*- coding: utf-8 -*-
+from abc import abstractmethod
+
+from scipy.stats import multivariate_normal
+
+from ..base import Base, Property
+from .kalman import KalmanUpdater
+from ..types.update import GaussianMixtureUpdate
+from ..types.state import TaggedWeightedGaussianState
+
+
+class PointProcessUpdater(Base):
+    r"""
+    """
+    updater = Property(
+        KalmanUpdater,
+        default=None,
```

Comment:
I don't think this can be `None`?

Reply:
I'm not sure of the relationship between the PointProcessUpdater (and its subclasses) and the PointProcessMultiTargetTracker. This may be my own limited understanding of the GMPHD type algorithms.